src/main/java/hudson/plugins/jira/listissuesparameter/JiraIssueParameterDefinition.java

```diff
@@ -98,7 +100,15 @@ public void setJiraIssueFilter(String jiraIssueFilter) {
         this.jiraIssueFilter = jiraIssueFilter;
     }
 
-    @Extension
+    public String getAltSummaryFields() {
```

Comment:
formatting is broken

Reply:
I'm sorry, I can't see what you are referring to?

Reply:
![image](https://user-images.githubusercontent.com/2369646/69595419-923ca780-1053-11ea-8c12-db23a84d5288.png)

Reply:
Fixed.  My editor was hiding it :(